website/docs/commands/data-structures.md

```diff
@@ -674,6 +674,19 @@ If **destination** already exists, it is overwritten.
 
 ---
 
+### SINTERCARD
+
+#### Syntax
+
+```bash
+    SINTERCARD key [key ...]
+```
+
+Returns the number of members in the resulting set from the intersection of all the given sets.

```

Comment:
You may need to update this when implementing the above optimization with LIMIT

Reply:
Done